### PR TITLE
added chains support

### PR DESCRIPTION
**Pull request by DFSko**
resolves happy-travel/agent-app-project#1750

### CODE REVIEW

File: HappyTravel.Komoro.Api/Services/Statics/IChainService.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+/// <summary>
+/// Interface for working with chains
+/// </summary>
+public interface IChainService
+{
+    /// <summary>
+    /// Returns list of chains
+    /// </summary>
+    /// <param name="cancellationToken"></param>
```
Comment (AlexeyShok) [inline HappyTravel.Komoro.Api/Services/Statics/IChainService.cs:15]: Need comments for parameters

File: HappyTravel.Komoro.Common/HappyTravel.Komoro.Common.csproj [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
   <ItemGroup>
     <PackageReference Include="CSharpFunctionalExtensions" Version="2.33.2" />
-    <PackageReference Include="HappyTravel.KomoroContracts" Version="0.6.0" />
+    <PackageReference Include="HappyTravel.KomoroContracts" Version="0.6.1" />
```
Comment (AlexeyShok) [inline HappyTravel.Komoro.Common/HappyTravel.Komoro.Common.csproj:12]: Current version in 0.6.2

**Review by AlexeyShok — APPROVED** [review]

File: HappyTravel.Komoro.Api/Services/Statics/IChainService.cs (hunk 1 above)
Comment (DFSko) [reply HappyTravel.Komoro.Api/Services/Statics/IChainService.cs]: added

File: HappyTravel.Komoro.Common/HappyTravel.Komoro.Common.csproj (hunk 2 above)
Comment (DFSko) [reply HappyTravel.Komoro.Common/HappyTravel.Komoro.Common.csproj]: fixed

**Pull request merged**